connectors/source-http/README.md

```diff
@@ -1,90 +1,177 @@
-# HTTP Source 
+# HTTP Source
 
-## Overview
+## Introduction
 
-A [Vance Connector][vc] which transforms HTTP requests to CloudEvents and deliver them to the target URL.
+The HTTP Source is a [Vance Connector](../README.md) which aims to convert incoming HTTP Request to a CloudEvent.
 
-## User Guidelines
+For example, if the incoming HTTP Request looks like:
 
-### Connector Introduction
-
-The HTTP Source is a [Vance Connector][vc] which aims to generate CloudEvents in a way that wraps all headers and body of the 
-original request into the `data` field of a new CloudEvent.
-
-For example, if an original request looks like:
-
-```http
-> POST /payload HTTP/2
-
-> Host: localhost:8080
-> User-Agent: VanceCDK-HttpClient/1.0.0
-> Content-Type: application/json
-> Content-Length: 39
-
-> {
->    "myData" : "simulation event data <1>"
-> }
+```bash
+curl --location --request POST 'localhost:8080/webhook?source=123&id=abc&type=456&subject=def' \
```

Comment:
do you think the params id, source, type, and subject use header is better?

Reply:
for most scenarios, users can't modify headers, because this is a webhook address, the requests aren't made by users. For example, GitHub only allows users to fill in an address to receive requests, and can't add headers.